api/google-search/src/main/java/com/google/search/googlesearch/controller/v1/api/SearchAutoCompleteController.java

```diff
@@ -0,0 +1,35 @@
+package com.google.search.googlesearch.controller.v1.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.google.search.googlesearch.models.SearchAutoCompleteModel;
+import com.google.search.googlesearch.services.SearchAutoCompleteService;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+
+@RestController
+@RequestMapping(value = "/api/v1/")
+public class SearchAutoCompleteController {
+	
+	@Autowired
+	private SearchAutoCompleteModel searchAutoCompleteModel;
+	
+	@Autowired
+	private SearchAutoCompleteService searchAutoCompleteService;
+	
+	@GetMapping(value = "search/autocomplete",produces = "application/json")
```

Comment:
Use constant instead of **produces = "application/json")**

Reply:
Updated with `MediaType.APPLICATION_JSON_VALUE`